src/utils/deploy.ts

```diff
@@ -92,15 +96,38 @@ export class DeployUtils {
 
     deploymentResponse.isSuccess
       ? PostHogUtil.trackEvent('DEPLOY_SUCCESS', {
-          msg: deploymentResponse.msg,
-        })
+        msg: deploymentResponse.msg,
+      })
       : PostHogUtil.trackEvent('DEPLOY_FAILED', {
-          error: deploymentResponse.msg,
-        });
+        error: deploymentResponse.msg,
+      });
 
     deploymentResponse.isSuccess
       ? console.log(deploymentResponse.msg)
       : console.error(deploymentResponse.msg);
+
+    if (deploymentResponse.isSuccess) {
+      const configFilePath = path.join(cwd, 'layerzero.config.ts');
```

Comment:
suggestion for the reference to these files to be outside of the logic , if someone changes the name it might be hard to catch